### ISSUE TITLE
FR: Volvo Trucks Magazine Feed

### ISSUE THREAD

**Issue by shirin27** [hlxsites/vg-volvotrucks-us#542]
### Description
As a business owner, I want to enhance the [Magazine feed of articles page](https://www.volvotrucks.us/news-and-stories/volvo-trucks-magazine/) to boost user engagement, increase page views, and enhance our brand's value by improving content discovery.

**Editor AC:**
- As an editor, I can create my magazine feed by using the following blocks: 
- [ ] https://github.com/hlxsites/vg-volvotrucks-us/issues/532 *In progress*
- [x] https://github.com/hlxsites/vg-volvotrucks-us/issues/619  *In progress*
- [x] https://github.com/hlxsites/vg-volvotrucks-us/issues/680
- [x] https://github.com/hlxsites/vg-volvotrucks-us/issues/681
- [ ] https://github.com/hlxsites/vg-volvotrucks-us/issues/602
- [x] https://github.com/hlxsites/vg-volvotrucks-us/issues/601
- [x] V2 Newsletter (existing block)

**Non-functional AC:**
- All of the new blocks above are added to the AEM Sidekick library. 

### Attachments/Figma Designs
**Desktop/Tablet:** https://www.figma.com/file/c2jG1bOcvle2bCmZqgptyS/Magazine-C%26T?type=design&node-id=1574%3A20612&mode=dev

**Mobile:** https://www.figma.com/file/c2jG1bOcvle2bCmZqgptyS/Magazine-C%26T?type=design&node-id=1597%3A22901&mode=dev

![Image](https://github.com/hlxsites/vg-volvotrucks-us/assets/125895595/0c96e4db-0a54-47e7-84fc-3857d10a1e41)

![Image](https://github.com/hlxsites/vg-volvotrucks-us/assets/125895595/0a494722-139c-473b-84b4-e386ccb00afa)

### Open Questions
- What type of input should this use to load more articles (load on scroll, load more button, pagination)? Answer: Pagination 
- What's the number of initial articles loaded? Answer: 12 on desktop/tablet, 8 on mobile
- How many articles are loaded on "load more"? Answer: no longer needed with pagination - see ticket https://github.com/hlxsites/vg-volvotrucks-us/issues/602
- Is it possible to automate the magazine feed so that when a new article is published, it would dynamically populate the magazine feed with a new content card block or something along those lines? (would create a new ticket for this if it's possible)

**Comment by shirin27** [comment]
@williebender Please see questions above.

**Comment by mollyaljenkins** [comment]
Blocked by search effort needed

**Comment by mollyaljenkins** [comment]
This ticket is a checklist and is not needed as development work. It can be closed as it will not be estimated, all blocks are estimated separately and will be added to the sidekick library as they are deployed.

**Comment by mollyaljenkins** [comment]
Closed as not needed for development work
